.gitlab-ci.yml

```diff
@@ -289,7 +289,7 @@ run_tests_windows-x86:
     - inv -e rtloader.install
     - inv -e rtloader.format --raise-if-changed
     - inv -e rtloader.test
-    - inv -e deps --verbose --dep-vendor-only
+    - inv -e deps --verbose --dep-vendor-only --release-version "$RELEASE_VERSION"
```

Comment:
Since this argument is only used for integrations, should we call it explicitly `integrations-version`? This would mean that if we ever download other things from `inv deps` we will have to add more flag for those things' versions, but I think I like it better (because it's more explicit).

Reply:
Changed!